### ISSUE TITLE
Since Ix-Async 1.2.4, its NuGet package cannot be added to WPA81 project

### ISSUE THREAD

**Issue by PaulosV** [Reactive-Extensions/Rx.NET#165]
Ix-Async can be installed to WPA81 (Windows Phone 8.1 - WinRT) without issue until 1.2.3. From 1.2.4, it is not possible anymore; the required assemblies for this are missing.
```
PM> Install-Package Ix-Async -Version 1.2.4 
Attempting to gather dependencies information for package 'Ix-Async.1.2.4' with respect to project 'XX.App\XX.App.WindowsPhone', targeting 'WindowsPhoneApp,Version=v8.1'
Attempting to resolve dependencies for package 'Ix-Async.1.2.4' with DependencyBehavior 'Lowest'
Resolving actions to install package 'Ix-Async.1.2.4'
Resolved actions to install package 'Ix-Async.1.2.4'
Install failed. Rolling back...
Package 'Ix-Async.1.2.4 : ' does not exist in project 'XX.App.WindowsPhone'
Package 'Ix-Async.1.2.4 : ' does not exist in folder 'I:\Documents\Projekty\XX\WindowsPhone\packages'
Install-Package : Could not install package 'Ix-Async 1.2.4'. You are trying to install this package into a project that targets 'WindowsPhoneApp,Version=v8.1', but the package does not contain any assembly references or content files that are compatible with that framework. 
For more information, contact the package author.
At line:1 char:1
+ Install-Package Ix-Async -Version 1.2.4
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
```

**Comment by Luke47** [comment]
Any news on this? :/

**Comment by onovotny** [comment]
#148 will run on Wpa81

**Comment by Luke47** [comment]
Thank you so much :)

**Comment by onovotny** [comment]
Will be fixed by 3.0, shipping on the 27th.
